src/components/ec-date-range-filter/ec-date-range-filter.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <ec-filter-popover
+    :label="label"
+    :number-of-selected-filters="numberOfSelectedFilters"
+    :popover-options="popoverOptions"
+  >
+    <template #filter>
+      <div
+        class="ec-date-range-filter"
+        data-test="ec-date-range-filter"
+      >
+        <div
+          class="ec-date-range-filter__inputs-wrapper"
+          data-test="ec-date-range-filter__inputs-wrapper"
+        >
+          <ec-input-field
+            v-model="fromValueDate"
+            class="ec-date-range-filter__from-label"
+            type="date"
+            placeholder="DD/MM/YYYY"
+            :label="fromLabelText"
+            :error-message="errorMessageFromDate"
+            @change="onChange()"
+            @input="errorMessageFromDate = ''"
+          />
+          <ec-input-field
+            v-model="toValueDate"
+            class="ec-date-range-filter__to-label"
+            type="date"
+            placeholder="DD/MM/YYYY"
+            :label="toLabelText"
+            @change="onChange()"
+            @input="errorMessageFromDate = ''"
+          />
+        </div>
+        <button
+          type="button"
+          :disabled="isDisabled"
+          class="ec-date-range-filter__clear-button"
+          data-test="ec-date-range-filter__clear-button"
+          @click="clearDates()"
+        >Clear Dates</button>
+      </div>
+    </template>
+  </ec-filter-popover>
+</template>
+
+<script>
+import EcInputField from '../ec-input-field';
+import ecFilterPopover from '../ec-filter-popover';
+
+export default {
+  name: 'EcDateRangeFilter',
+  components: { ecFilterPopover, EcInputField },
+  props: {
+    label: {
+      type: String,
+      required: true,
+      default: '',
+    },
+    value: { // Check we actually need this
```

Comment:
do we? AFAIK, we need the separate from and to values, and not something we'll compose for later use, probably we can get rid of this